salt/etcd/init.sls

```diff
@@ -32,10 +32,6 @@ etcd:
       - etcd
     - require:
       - file: /etc/zypp/repos.d/containers.repo
-  cmd.run:
-    - name: rm -rf /var/lib/etcd/*
-    - prereq:
-      - service: etcd
```

Comment:
We are currently tracking this issue in [this card](https://trello.com/c/2WxhX4hX)